core/templates/components/summary-tile/story-summary-tile.component.html

```diff
@@ -215,7 +215,7 @@
           <div *ngIf="node.getStatus() === 'Published' && storySummary.isNodeCompleted(node.getTitle())" class="fas fa-check completed-icon">
           </div>
           <div [ngClass]="{'pending-chapter': !storySummary.isNodeCompleted(node.getTitle()) && isPreviousChapterCompleted(idx), 'incomplete-chapter': !storySummary.isNodeCompleted(node.getTitle()), 'complete-chapter': storySummary.isNodeCompleted(node.getTitle())}"
-              *ngIf="node.getStatus() === 'Published'">
+              *ngIf="node.getStatus() === 'PubligetChapterUrlshed'">
```

Comment:
Unexpected changes? (Make sure to go through the `Files changed` tab after pushing new commits.)